action.yml

```diff
@@ -100,10 +100,10 @@ runs:
     - run: |
         : enable Cargo sparse registry
         # except on 1.66 and 1.67, on which it is unstable
-        if [ -z ${CARGO_REGISTRIES_CRATES_IO_PROTOCOL+set} ] \
-          && ! rustc +${{steps.parse.outputs.toolchain}} --version --verbose | grep -q '^release: 1\.6[67]\.'
-        then
-          echo CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse >> $GITHUB_ENV
+        if [ -z ${CARGO_REGISTRIES_CRATES_IO_PROTOCOL+set} ]; then
+          if rustc +${{steps.parse.outputs.toolchain}} --version --verbose | not grep -q '^release: 1\.6[67]\.'; then
```

Comment:
It may also need to check `commit-date` since it's only stabilized after 2023-01-20 or later: https://github.com/rust-lang/rust/pull/107120, but the rustc release was already at 1.68.

See https://github.com/pola-rs/polars/actions/runs/4183969359/jobs/7249008618